index.js

```diff
@@ -214,6 +247,7 @@ function Polyglot(options) {
   this.warn = opts.warn || warn;
   this.tokenRegex = constructTokenRegex(opts.interpolation);
   this.pluralRules = opts.pluralRules || defaultPluralRules;
+  this.knownLocalePluralTypeName = {};
```

Comment:
It’s unfortunate to expose yet another thing on the instance; can that be avoided?

Reply:
Thanks, I overlooked that.